test/unit/array.spec.ts

```diff
@@ -191,3 +191,12 @@ test.each([
     `;
     expect(() => util.transpileAndExecute(code)).toThrowError(`invalid array length: ${result}`);
 });
+
+test("Array OmittedExpression", () => {
+    const result = util.transpileAndExecute(
+        `const myarray = [1, , 2];
+        return myarray[2];`
+    );
+
+    expect(result).toBe(2);
```

Comment:
I think this test should check the result of the omittedexpression, not the expression after it (so check myarray[1] == undefined). I'm fine with also testing the elements after like you do now, either as additional expect in this test, or as a separate test.